test/functional/feature_maxtipage.py

```diff
@@ -51,6 +52,9 @@ def run_test(self):
             self.log.info(f"Test IBD with maximum tip age of {hours} hours (-maxtipage={maxtipage}).")
             self.test_maxtipage(maxtipage)
 
+        self.log.info(f"Test IBD with max. allowable maximum tip age ({maxtipage}).")
```

Comment:
This is printing a value that went out of scope (maybe we should use rust to write tests?)

Also, the value is not the max, as claimed here.

You can take commit 0d3b7631af03ecc4d6b9e56a74ac2abfe8977291, where I fixed this.

Reply:
Fixed, thanks.